src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -6,10 +6,10 @@
         const { onShow, label, message, rawPlaceholdersMap = {} } = detail;
         // @TODO: Unify 'error' and 'danger' label in 3.0.
         const configKey = label === 'danger' ? 'error' : label;
-        const config = eZ.adminUiConfig.notifications[configKey];
+        const config = eZ.adminUiConfig ? eZ.adminUiConfig.notifications[configKey] : null;
         const timeout = config ? config.timeout : 0;
         const container = doc.createElement('div');
-        let finalMessage = eZ.helpers.text.escapeHTML(message);
+        let finalMessage = eZ.helpers.text ? eZ.helpers.text.escapeHTML(message) : message;
```

Comment:
This was added for security reasons. Don't think we should ever bypass this.

Reply:
In order to have any helpers in `Security/base.html.twig` I should implement the [addConfig](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/views/layout.html.twig#L20) function. Would it be okay if I separate this code into other `twig` file and include it in both files `Security/base.html.twig` and `layout.html.twig`?